bin/webpack.js

```diff
@@ -166,9 +166,11 @@ if(argv.init) {
 	if (!filePaths.length) {
 		throw new Error('Please specify a path to your webpack config');
 	}
+	const outputConfigName = filePaths[1] || `${filePaths[0]}v2.js`;
```

Comment:
This will make git diff impossible and it will require an extra step from users to remove the old config and rename the new. IMO it defeats the purpose of using this tool. Why would someone like to keep the old config?